src/test/scala/uk/gov/homeoffice/drt/model/DepartureAirportSpecs.scala

```diff
@@ -16,34 +16,34 @@ class DepartureAirportSpecs extends AsyncFlatSpec with Matchers {
   }
 
   "Departure Airport code" should "give list of all airport for Croatia" in {
-    val greeceList = DepartureAirport.athensDeparturePortsForCountry("Croatia")
+    val greeceList = DepartureAirport.tiranaDeparturePortForCountry("Croatia")
```

Comment:
maybe rename the `greeceList` to something generic & descriptive in each of these test, eg `airports`